### PR TITLE
Adds styles on the editor for alignleft/alignright. Fixes #282

### LINKED ISSUE #282: Update design of floated pullquotes
Moving this over from #229:

> Do we want to limit the width of the left and right aligned pullquote to 50%? When the content is long enough, the quotes are no longer aligned but looks like they are normal width without alignments.
>
> In the editor, left and right aligned pull quotes needs spacing to the right and left of the block respectively.
>
> It is less visible on the right:

image

### PR DESCRIPTION
**Pull request by aristath**
Fixes #282 - Adds styles on the editor for alignleft/alignright

### CODE REVIEW

File: assets/sass/05-blocks/utilities/_editor.scss [hunk 1]
```diff
@@ -179,3 +179,16 @@
 	margin: 0;
 	margin-left: var(--global--spacing-horizontal);
 }
+
+@media only screen and (min-width: 482px) {
+
+	.wp-block[data-align="left"] > * {
+		max-width: 290px;
```
Comment (kjellr) [inline assets/sass/05-blocks/utilities/_editor.scss:186]: Can we use a variable here, rather than hardcoding this width?

**Comment by carolinan** [comment]
Looks good to me, tested several block types.

**Review by carolinan — APPROVED** [review]

File: assets/sass/05-blocks/utilities/_editor.scss (hunk 1 above)
Comment (aristath) [reply assets/sass/05-blocks/utilities/_editor.scss:186]: I tried the same same calc we use on the front-end but there were lots of issues with it. Hardcoding a value seemed like a reasonable compromise.
We don't have any related vars that I could see, and adding a new one just for the purpose of using it here didn't seem right

Comment (kjellr) [reply assets/sass/05-blocks/utilities/_editor.scss:186]: Maybe we use something based on the [sass variable](https://github.com/WordPress/twentytwentyone/blob/07eab795ee6c80c662293d800081af22c39facac/assets/sass/03-generic/breakpoints.scss#L9) then? Something like `#{0.5 * $default_width}`.

**Comment by carolinan** [comment]
Because we need the alignments for testing the blocks, I am going to merge this without the CSS variable change.

**Pull request merged**